src/store/engine.ts

```diff
@@ -31,6 +31,26 @@ export const engineStore = createPartialStore<EngineStoreTypes>({
     },
   },
 
+  UPDATE_ENGINE_INFO: {
```

Comment:
UPDATEのままでもほぼ問題ないのですが、同じようなことをするのは全部SETになっていそうなので、SETにするのはどうでしょう。
```suggestion
  SET_ENGINE_INFO: {
```